plotly/tools.py

```diff
@@ -1501,6 +1508,35 @@ def validate_ohlc(open, high, low, close, direction, **kwargs):
                                          "'both'")
 
     @staticmethod
+    def validate_distplot(hist_data, curve_type):
```

Comment:
can we make these guys hidden? (by adding a `_`). there's hardly anything I like more than seeing the minimal number of functions in the ipython tab-tab helper:
![image](https://cloud.githubusercontent.com/assets/1280389/9555375/a432ba36-4d9a-11e5-8d58-97095f0eedb0.png)